code-of-conduct.html

```diff
@@ -0,0 +1,134 @@
+<html>
+<head>
+<title>satRday Code of Conduct</title>
+</head>
+<body>
+<h1 align="center"> satRday Code of Conduct/ Anti-Harassment Policy </h1>
```

Comment:
I'm not quite sure where my fault is with this... 

I've applied the CSS stylesheet but when i remove "text-align: center;" in the h1 tag it moves back to the left? (All the headings are in fact on the left aside from the h1-after the css stylesheet addition.)

Have removed the style sheet ref's and then reapplied them (copy and pasting from here) and it still seems to have the same affect...
![selection_024](https://user-images.githubusercontent.com/35062730/34714128-19699f32-f531-11e7-8450-e75180e2afc5.png)

Reply:
I'm not entirely sure, but I'm guessing that maybe you are testing by rendering this page statically in a browser in your local environment and not via a Jekyll environment (e.g. `jekyll serve`)?

In this case it's likely that `css/main.css` can't be linked since it doesn't exist (my limited understanding is that `css/main.css` is generated from `css/main.scss` in the Jekyll context via a CSS pre-processor. You can verify this by inspecting the page once it has rendered and seeing what CSS styles are being inherited by h1, h2 etc.

Sorry if I've over-complicated things for you here - I blame it on the codebase I inherited from @DataWookie 😉 

I'm pretty sure you can just trust that it will work since I tested this locally last night when I made the suggestions, but it's possible my explanation above for why you're seeing something different is incorrect. I'm just making sense of things as I go through trial and error since I actually don't know much about CSS or web development in general. I am a complete fraud 🤷‍♂️ 

If you want to you can still embed/specify each necessary bit of CSS in the html itself to make it more clear - I was taking the somewhat lazy approach of leveraging the CSS that exists already, even though it is kinda overkill for a simple page like this one. But I suppose it's justified given that the guiding principle should generally be to try and avoid custom CSS all over the place so that when things need to be updated going forward that only has to be done in one or two key places rather than all over the place. The downside in this case is that you'll probably need to install and setup Jekyll in order to render and test the site locally.